transactions/epoch/admin/pay_rewards.cdc

```diff
@@ -6,6 +6,8 @@ transaction {
         let heartbeat = signer.borrow<&FlowEpoch.Heartbeat>(from: FlowEpoch.heartbeatStoragePath)
             ?? panic("Could not borrow heartbeat from storage path")
 
-        heartbeat.payRewards()
+        let previousEpochMetadata = FlowEpoch.getEpochMetadata(FlowEpoch.currentEpochCounter - (1 as UInt64))!
```

Comment:
Sounds like you understand how it works now, right? We don't actually use this transaction right now, but if we did, we would run it after an epoch ends, not before. This transaction is kind of an in-between step between how we do it now (manually using only the staking contract) and automated rewards (using the epoch contract)

Reply:
Do you have any other problems with this transaction or can it remain as-is?

Reply:
> We don't actually use this transaction right now, but if we did, we would run it after an epoch ends, not before

Thanks, yes I understand how we would change the manual reward payment transaction timing now.

From my perspective, the fact that sending the same manual `PayRewards` transaction at slightly different times can result in different outcomes, depending on the current epoch state, is prone to errors. This property is a big part of the recent manual reward payout issue we've been discussing in Slack. 

The proposed `epoch/admin/pay_rewards.cdc` transaction in this PR  maintains the property of timing dependency (just different timing). However:
* if we accidentally send the transaction early (before the epoch switchover), it will fail rather than pay the wrong rewards
* after the epoch switchover, we have 1 week to get the transaction in, rather than only a few hours with the current manual reward payout

So overall I think this in-between manual payout transaction is addressing most of the drawbacks of the one we currently use. Happy to keep it as-is 👍